e2e-tests/cypress/e2e/journeys/insurance/your-buyer/validation/company-or-organisation-validation-address.spec.js

```diff
@@ -0,0 +1,77 @@
+import { submitButton } from '../../../../pages/shared';
+import { companyOrOrganisationPage } from '../../../../pages/insurance/your-buyer';
+import partials from '../../../../partials';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import { ROUTES } from '../../../../../../constants';
+import { YOUR_BUYER as FIELD_IDS } from '../../../../../../constants/field-ids/insurance/your-buyer';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const {
+  COMPANY_OR_ORGANISATION: {
+    ADDRESS,
+  },
+} = FIELD_IDS;
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: {
+      COMPANY_OR_ORGANISATION: COMPANY_OR_ORG_ERROR_MESSAGES,
+    },
+  },
+} = ERROR_MESSAGES;
+
+const goToPageDirectly = (referenceNumber) => {
```

Comment:
Do we need this function?

Reply:
Good shout - initially just used the same approach as other "your buyer" E2E tests. i've removed this function and updated the tests to click through via the task list, like the real world user journey